docs/source/usage/graders/writing_graders.rst

```diff
@@ -0,0 +1,175 @@
+################
+Writing a Grader
+################
+
+.. caution::
+
+   It isn't as simple as it sounds - there are certain traps
+   that are easy to fall into. Read the full page before writing a grader script.
+
+Tin allows you to use the full flexibility of Python (or Java)
```

Comment:
I don't think the `(or Java)` addition here is correct, since although there may be workarounds to use Java for grading, currently Tin doesn't really "allow" you to do that (uploaded graders must be in Python...)